.github/workflows/deploy-wallet-at-pr.yml

```diff
@@ -186,25 +122,12 @@ jobs:
         working-directory: ./apps/extension
         run: yarn build:chrome
 
-      # GitHub actions zips artifacts so we need to unzip the built extension to
-      # avoid a double zip
-      - name: Prepare artifact
```

Comment:
If I understood correctly, @jurevans use this to build the extension and share with the team

Reply:
It is actually this one that is used for the extension build:
https://github.com/anoma/namada-interface/actions/workflows/release-wallet.yml

As far as I know the artifacts are not used from this workflow, so I removed them.